infra/base-images/base-libfuzzer/reproduce

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eux
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FUZZER=$1
+shift
+TESTCASE="/testcase"
+
+if [ ! -f $TESTCASE ]; then
+  echo "Error: $TESTCASE not found, use: docker run -v <path>:$TESTCASE ..."
+  exit 1
+fi
+
+
+compile
+export PATH=/out:$PATH
+cd /out
+
+TEST_CORPUS=$(tempfile)
+rm -rf $TEST_CORPUS
+mkdir $TEST_CORPUS
+cp $TESTCASE $TEST_CORPUS/
+$FUZZER $@ -runs=0 $TEST_CORPUS
```

Comment:
(re-adding comment, looks like my other one was hidden by default)

why do we need a directory? can we just pass the testcase path directly?